### PR TITLE
Linted files outside of `/app`

### PR DESCRIPTION
**Pull request by Greenheart**
**Note:** This doesn't fix the temporary linting-errors introduced in PR #106
since they will be fixed when that feature is completed.

### CODE REVIEW

File: web/config.py [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import os
 
 
-basedir = os.path.abspath(os.path.dirname(__file__))
```
Comment (Greenheart) [inline web/config.py:4]: unused variable

**Comment by Limpan** [comment]
Can't the remaining errors be fixed too? It would be great to let Travis do something useful.

**Comment by Greenheart** [comment]
@Limpan All of them, or just the "normal" errors from `manage.py lint`?

**Comment by Limpan** [comment]
All of them. Or at least the ones in the tests directory. That way #130 won't be a pain.

**Comment by Greenheart** [comment]
@Limpan The rest of the linting-errors comes from `/migrations/` and `/docs/`.
- Should we ignore `migrations`-directory completely?
- Is the conf-file in `docs` something we've created or is that too auto-generated --> If automatic, let's ignore `/docs/` too?

Apart from those questions, this is ready to be merged! :smile:

**Pull request merged**